rules/prefer-math-trunc.js

```diff
@@ -33,6 +39,23 @@ const createBitwiseNotSelector = (level, isNegative) => {
 	return isNegative ? `:not(${selector})` : selector;
 };
 
+// All binary expressions
+const binarySelectors = [
```

Comment:
Listen to `BinaryExpression` and `AssignmentExpression` selector, check `operator` and `right` inside,

```js

const operators = new Map([
	['|', 0],
	// ...
]);

'BinaryExpression[right.type="Literal"]'(node) {
	const {operator, right} = node;
	if (operators.has(node.operator) && 
		typeof right.value === 'number' && 
		right.value === operators.get(node.operator)
	) {
		// ...
	}
}
```

Reply:
I'll do it but why do you want that? To have a more readable code? Is it faster?

Reply:
It's more flexible, 

```js
const operators = new Map([
	['|', 0],
	// ...
	['SOME_NEW_OPERATOR', A_NEW_VALUE]
]);
```

Reply:
Image if we already doing this, we only need add 

```js
	['<<', 0],
	['>>', 0],
```

All done.